packages/edit-site/src/components/sidebar-navigation-screen-templates-browse/content.js

```diff
@@ -25,7 +26,7 @@ function TemplateDataviewItem( { template, isActive } ) {
 
 	return (
 		<SidebarNavigationItem
-			to={ `/template?activeView=${ text }` }
+			to={ addQueryArgs( '/template', { activeView: text } ) }
```

Comment:
I don't undestand what's the difference between the two codes?

Reply:
Basically, URL encoding. 

The version in trunk will navigate to:

```
/template?activeView=Twenty Twenty-Four
```

while the version here will navigate to:

```
/template?activeView=Twenty%20Twenty-Four
```

and the former seems to break the navigation.

Reply:
makes sense :)

Reply:
Shipping this one, but it might make sense to double-check other instances where we hardcoded the query string instead of using `addQueryArgs()`. 

Cases like this are exactly why I was suggesting defaulting to `addQueryArgs()` FWIW.